src/implementations/twig/components/footer-standardised/footer-standardised.story.js

```diff
@@ -1,190 +1,140 @@
-              `sections[${i}].links[${j}].icon.name`,
-              linkItem.icon.name,
-              tabLabels.required
-            );
-            linkItem.icon.path = text(
-              `sections[${i}].links[${j}].icon.path`,
-              defaultSprite,
-              tabLabels.required
-            );
-            linkItem.icon.size = text(
-              `sections[${i}].links[${j}].icon.size`,
-              linkItem.icon.size,
-              tabLabels.required
-            );
-          }
-        });
-      }
-    } else {
-      section.forEach((sectionItem, j) => {
-        if (sectionItem.content_before) {
-          sectionItem.content_before = text(
-            `sections[${i}][${j}].content_before`,
-            sectionItem.content_before,
-            tabLabels.optional
-          );
-        }
-        if (sectionItem.list_class_name) {
-          sectionItem.list_class_name = text(
-            `sections[${i}][${j}].list_class_name`,
-            sectionItem.list_class_name,
-            tabLabels.optional
-          );
-        }
-        if (sectionItem.links) {
-          sectionItem.title = text(
-            `sections[${i}][${j}].title`,
-            sectionItem.title,
-            tabLabels.optional
-          );
-          sectionItem.links.forEach((linkItem, k) => {
-            linkItem.link.label = text(
-              `sections[${i}][${j}].links[${k}].link.label`,
-              linkItem.link.label,
-              tabLabels.optional
-            );
-            linkItem.link.path = text(
-              `sections[${i}][${j}].links[${k}].link.path`,
-              linkItem.link.path,
-              tabLabels.optional
-            );
-            linkItem.link.aria_label = text(
-              `sections[${i}][${j}].links[${k}].link.aria_label`,
-              linkItem.link.aria_label,
-              tabLabels.optional
-            );
-            if (linkItem.icon) {
-              linkItem.icon.name = text(
-                `sections[${i}][${j}].links[${k}].icon.name`,
-                linkItem.icon.name,
-                tabLabels.optional
-              );
-              linkItem.icon.path = text(
-                `sections[${i}][${j}].links[${k}].icon.path`,
-                defaultSprite,
-                tabLabels.optional
-              );
-              linkItem.icon.size = text(
-                `sections[${i}][${j}].links[${k}].icon.size`,
-                linkItem.icon.size,
-                tabLabels.optional
-              );
-            }
-          });
-        }
-      });
-    }
-  });
+const getArgTypes = (system) => {
+  const argTypes = {};
+  argTypes.hide_contact = {
+    name: system === 'EU' ? 'contact site name' : 'contact us',
+    type: { name: 'boolean' },
+    defaultValue: false,
+    description:
+      system === 'EU'
+        ? 'Hide "Contact site name" section'
+        : 'Hide "Contact us" section',
+    table: {
+      type: { summary: 'boolean' },
```

Comment:
mmh..ok that we define these controls as boolean to have a checkbox, but there is no relation with the data in this case, i would simply leave the table "empty", no..? No default value, no "type"